modules/monitoring-support-version-matrix-for-monitoring-components.adoc

```diff
@@ -10,9 +10,9 @@ The following matrix contains information about versions of monitoring component
 
 .{product-title} and component versions
 |===
-|{product-title} |Prometheus Operator |Prometheus  |Prometheus Adapter |Metrics Server (Technology Preview) |Alertmanager |kube-state-metrics agent |monitoring-plugin |node-exporter agent |Thanos
+|{product-title} |Prometheus Operator |Prometheus  |Prometheus Adapter |Metrics Server |Alertmanager |kube-state-metrics agent |monitoring-plugin |node-exporter agent |Thanos
```

Comment:
since metrics-server is GA and replaces Prometheus Adapter, then we should remove `Prometheus Adapter`

Reply:
@juzhao will `Prometheus Adapter` be already removed in 4.16, or just still deprecated?

Reply:
@eromanova97 `Prometheus Adapter` is replaced by  metrics-server, no `Prometheus Adapter` pod now
```
$ oc -n openshift-monitoring get pod | grep -E "prometheus-adapter|metrics-server"
metrics-server-ccb58b6dc-2zn9n                           1/1     Running   0          109m
metrics-server-ccb58b6dc-kxg2j                           1/1     Running   0          109m
```

Reply:
Great, I will remove it then :+1: